actions/workflows/st2_prep_release_rules.yaml

```diff
@@ -0,0 +1,71 @@
+version: '2.0'
+st2cd.st2_prep_rules:
+    description: Prepare st2cd and st2ci rules for next release.
+    input:
+        - version
+        - prev_version
+        - host
+        - cwd
+
+    task-defaults:
+        on-error:
+            - cleanup_on_failure: <% $.local_repo_sfx != null %>
```

Comment:
Can you add a vars section and init local_repo_sfx to null?  Otherwise, if there's failure with init task before the publish, this task-defaults will fail.

```
    input:
        ...

    vars:
        local_repo_sfx: null

    task-defaults:
        ...
```